charts/incubator/dendrite/values.yaml

```diff
@@ -0,0 +1,246 @@
+                echo "      method: scale" >> /etc/dendrite/dendrite.yaml
+                echo "" >> /etc/dendrite/dendrite.yaml
+                echo "mscs:" >> /etc/dendrite/dendrite.yaml
+                echo "  mscs:" >> /etc/dendrite/dendrite.yaml
+                echo "#    - msc2836" >> /etc/dendrite/dendrite.yaml
+                echo "" >> /etc/dendrite/dendrite.yaml
+                echo "sync_api:" >> /etc/dendrite/dendrite.yaml
+                echo "  real_ip_header: X-Real-IP" >> /etc/dendrite/dendrite.yaml
+                echo "" >> /etc/dendrite/dendrite.yaml
+                echo "  search:" >> /etc/dendrite/dendrite.yaml
+                echo "    enabled: $DENDRITE_ENABLE_SEARCH" >> /etc/dendrite/dendrite.yaml
+                echo ""
+                echo "    index_path: /var/dendrite/searchindex" >> /etc/dendrite/dendrite.yaml
+                echo "" >> /etc/dendrite/dendrite.yaml
+                echo "    language: $DENDRITE_SEARCH_LANG" >> /etc/dendrite/dendrite.yaml
+                echo "" >> /etc/dendrite/dendrite.yaml
+                echo "user_api:" >> /etc/dendrite/dendrite.yaml
+                echo "  bcrypt_cost: $DENDRITE_USER_BCRYPT_COST" >> /etc/dendrite/dendrite.yaml
+                echo ""  >> /etc/dendrite/dendrite.yaml
+                echo "  auto_join_room:" >> /etc/dendrite/dendrite.yaml
+                echo "#" >> /etc/dendrite/dendrite.yaml
+                echo "" >> /etc/dendrite/dendrite.yaml
+                echo "tracing:" >> /etc/dendrite/dendrite.yaml
+                echo "  enabled: false" >> /etc/dendrite/dendrite.yaml
+                echo "" >> /etc/dendrite/dendrite.yaml
+                echo "logging:" >> /etc/dendrite/dendrite.yaml
+                echo "  - type: std" >> /etc/dendrite/dendrite.yaml
+                echo "    level: info" >> /etc/dendrite/dendrite.yaml
+                echo "  - type: file" >> /etc/dendrite/dendrite.yaml
+                echo "    level: info" >> /etc/dendrite/dendrite.yaml
+                echo "    params:" >> /etc/dendrite/dendrite.yaml
+                echo "      path: ./logs" >> /etc/dendrite/dendrite.yaml
+                echo "##########################################################" >> /etc/dendrite/dendrite.yaml;
+                echo 'Initial configuration file created at "/etc/dendrite/dendrite.yaml"';
+              fi;
+      containers:
+        main:
+          probes:
+            liveness:
+              type: tcp
+              port: /
+            readiness:
+              type: tcp
+              port: /
+            startup:
+              type: tcp
+              port: /
+
+persistence:
+  config:
+    enabled: true
+    mountPath: /etc/dendrite
+  media:
+    enabled: true
+    mountPath: /var/dendrite/media
+  jetstream:
+    enabled: true
+    mountPath: /var/dendrite/jetstream
+  dendrite_search_index:
+    enabled: true
+    mountPath: /var/dendrite/searchindex
+
+    # Enabled postgres
```

Comment:
Comment be like 
![image](https://github.com/truecharts/charts/assets/47820033/27674535-5b57-4d81-a49a-bea9e319838d)

Reply:
Lmao, yeah I just need an idea on what to do with the configmap/init containers, like do I create one using the zigbee2mqtt way or copy the sample, rename and then try to insert values, I'm pretty lost haha